<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LancerDeClass</class>
 <widget class="QMainWindow" name="LancerDeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LancerDe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dé</string>
     </attribute>
     <widget class="QPushButton" name="AfficheVal">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher Valeur Dé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="TireDe">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Tiré Dé</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Reset">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>210</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Remise 0 Score</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AfficheValTotal">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Afficher Score Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="ScoreTotal">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ChiffreObtenue">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>1011</width>
        <height>651</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QPushButton" name="AfficheVal_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher Valeur Dé</string>
        </property>
       </widget>
       <widget class="QPushButton" name="TireDe_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>80</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Tiré Dé</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Reset_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>340</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise 0 Score</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AfficheValTotal_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>231</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher Score Total</string>
        </property>
       </widget>
       <widget class="QLabel" name="ScoreTotal_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>400</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="ChiffreObtenue_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>180</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>21</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Historique Dé</string>
     </attribute>
     <widget class="QLineEdit" name="nbrDe">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>430</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="ChoixNbrDe">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>350</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Choisir le nombre de dé a lancer</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="plainText">
       <string>Maximum de lancer simultanné : 7</string>
      </property>
     </widget>
     <widget class="QLabel" name="Erreur">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>530</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>685</x>
        <y>40</y>
        <width>291</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LancerDe.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AfficheVal</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>AfficheValeurDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TireDe_2</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>TireDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AfficheValTotal</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>AfficheScoreTotal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Reset_2</sender>
   <signal>clicked()</signal>
   <receiver>LancerDeClass</receiver>
   <slot>Remise0Score()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChoixNbrDe</sender>
   <signal>clicked(bool)</signal>
   <receiver>LancerDeClass</receiver>
   <slot>TireMultiDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AfficheValeurDe()</slot>
  <slot>TireDe()</slot>
  <slot>AfficheScoreTotal()</slot>
  <slot>Remise0Score()</slot>
  <slot>TireMultiDe()</slot>
 </slots>
</ui>
